internal/emval.go

```diff
@@ -1234,8 +1235,97 @@ var EmvalEquals = api.GoModuleFunc(func(ctx context.Context, mod api.Module, sta
 })
 
 var EmvalGetModuleProperty = api.GoModuleFunc(func(ctx context.Context, mod api.Module, stack []uint64) {
-	// @todo: implement me.
-	panic("EmvalGetModuleProperty call unimplemented")
+	engine := MustGetEngineFromContext(ctx, mod).(*engine)
+	namePtr := api.DecodeI32(stack[0])
+	name, err := engine.getStringOrSymbol(uint32(namePtr))
+	if err != nil {
+		panic(fmt.Errorf("could not get symbol name"))
+	}
+
+	switch name {
+	case "HEAP8":
+		typedMemoryView, ok := allMemoryAs[int8](mod.Memory(), 1)
+		if !ok {
+			panic(fmt.Errorf("could not create memory view"))
+		}
+		stack[0] = api.EncodeI32(engine.emvalEngine.toHandle(js.Int8Array{
```

Comment:
We need to pass pointers for all the structs. Otherwise it fails with:

```
error while calling embind function full_default: reflect: Elem of invalid type js.Uint8Array (recovered by wazero)
```